OpenRA.Mods.RA/Traits/SupportPowers/ParatroopersPower.cs

```diff
@@ -204,9 +204,14 @@ public Actor[] SendParatroopers(Actor self, WPos target, bool randomize = true,
 					beacon = new Beacon(
 						self.Owner,
 						target - new WVec(0, 0, altitude),
-						Info.BeaconPalettePrefix,
+						Info.BeaconPaletteIsPlayerPalette,
+						Info.BeaconPalette,
+						Info.BeaconImage,
 						Info.BeaconPoster,
 						Info.BeaconPosterPalette,
+						Info.CircleSequence,
+						Info.ArrowSequence,
```

Comment:
You've got the order of the arguments flipped around here: `Info.ArrowSequence` comes before `Info.CircleSequence`.